drv/hisi_qm_udrv.c

```diff
@@ -251,12 +260,24 @@ static int his_qm_set_qp_ctx(handle_t h_ctx, struct hisi_qm_priv *config,
 	q_info->qc_type = qp_ctx.qc_type;
 	ret = wd_ctx_set_io_cmd(h_ctx, UACCE_CMD_QM_SET_QP_CTX, &qp_ctx);
 	if (ret < 0) {
-		WD_DEV_ERR(h_ctx, "failed to set qc_type, use default value!\n");
+		WD_DEV_ERR(h_ctx, "failed to set qc_type!\n");
 		return ret;
 	}
-
 	q_info->sqn = qp_ctx.id;
 
+	ret = wd_ctx_set_io_cmd(h_ctx, UACCE_CMD_QM_SET_QP_INFO, &qp_cfg);
+	if (ret < 0) {
+		WD_DEV_ERR(h_ctx, "set qp info failed, use default value!\n");
```

Comment:
This log will be flushed frequently, it is recommended to modify it

Reply:
modify to WD_INFO